save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/cvsscalculator/VectorTypes.kt

```diff
@@ -77,5 +82,19 @@ enum class CiaType(val value: String) {
     HIGH("H"),
     LOW("L"),
     NONE("N"),
+    NOT_DEFINED("_"),
+    ;
+}
+
+/**
+ * Version of CVSS
+ *
+ * @property value abbreviated value from the cvss version
+ */
+@Serializable
+enum class CvssVersion(val value: String) {
+    FOUR("4.0"),
```

Comment:
it's not released yet, suggestion to no support now

Reply:
Removed